docs/network-flow-visibility.md

```diff
@@ -0,0 +1,101 @@
+# Network Flow Visibility
+## Purpose
```

Comment:
As this document talks about the purpose and details of ELK flow collector, the title "Elk Flow Collector" is appropriate. This is can be a section in the document like "Flow Exporter": https://github.com/vmware-tanzu/antrea/pull/980/files#diff-a27d53d71ee3ed485cf12d6147782edb